api/src/routers/company.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlmodel import Session
+from src.classes.company import CompanyRepresentation
+from src.classes.pagination import CompanyPagination
+from src.db import get_session
+from src.models import CompanyCreate, CompanyPublic, CompanyUpdate
+
+router = APIRouter()
+
+
+@router.post("/company")
+async def create_company(data: CompanyCreate, session: Session = Depends(get_session)) -> None:  # noqa: B008
+    """Endpoint to create a new Company.
+
+    :param data: Company data.
+    :param session: Database session.
+    :return: Success or not.
+    """
+    # Creating the company
+    _ = CompanyRepresentation.create_company(session=session, data=data)
```

Comment:
Could the `_ = ` be removed?

Reply:
Will have a check. Probably could be. If so, that is definitely neater.